static/js/components/search/LearnerResult.js

```diff
@@ -88,23 +59,4 @@ class LearnerResult extends SearchkitComponent {
   }
 }
 
-const mapStateToProps = state => {
-  return {
-    learnerChipVisibility: state.ui.learnerChipVisibility
-  }
-}
-
-const mapDispatchToProps = (
-  dispatch: Dispatch,
-  ownProps: LearnerResultProps
-) => {
-  return {
-    setLearnerChipVisibility: (username: ?string): void => {
-      if (ownProps.learnerChipVisibility !== username) {
-        dispatch(setLearnerChipVisibility(username))
-      }
-    }
-  }
-}
-
-export default connect(mapStateToProps, mapDispatchToProps)(LearnerResult)
+export default connect()(LearnerResult)
```

Comment:
do we still need `connect()` here? It looks like we're no longer depending on anything from the Redux store